nemo_text_processing/g2p/modules.py

```diff
@@ -280,13 +280,7 @@ def __init__(
         Args:
             phoneme_dict (str, Path, Dict): Path to file in CMUdict format or dictionary of CMUdict-like entries.
                 Must be given for IPA G2P. (Consider using scripts/tts_dataset_files/ipa_cmudict-0.7b_nv22.06.txt.)
-            word_tokenize_func: Function for tokenizing text to words.
-                It has to return List[Tuple[Union[str, List[str]], bool]] where every tuple denotes word
-                representation and flag whether to leave unchanged or not.
-                It is expected that unchangeable word representation will be represented as List[str], other
-                cases are represented as str.
-                It is useful to mark word as unchangeable which is already in phoneme representation.
-                Defaults to the English word tokenizer.
```

Comment:
Worth moving this info down to the added section?

Reply:
This is very similar to the comment in the data_utils._word_tokenize() method itself. Can we assume that anyone who wants to override the behavior will look at the original function as a reference?

Though it would make sense to me to add at least the first line (describing what the return value is) somewhere in the tokenizer.

Reply:
Yeah, makes sense.

> Though it would make sense to me to add at least the first line (describing what the return value is) somewhere in the tokenizer.

This would be nice!